.python-version

```diff
@@ -1 +1 @@
-3.6.7
+3.6.0
```

Comment:
Why change the python version? If you were getting the error that version 3.6.7 doesn't exist, you need to do a git pull.

`cd /Users/YOUR_USERNAME_HERE/.pyenv/plugins/python-build/../.. && git pull`
^ taken from what I got when I tried to install 3.6.7 and it couldn't find it

Reply:
That was probably a side effect of trying to fix the error I was getting during initial startup. I just reverted back.